glados-web/assets/css/content_dashboard.css

```
@@ -0,0 +1,53 @@
+.dashboard-box {
+    background-color: #ffffff;
+    padding: 20px;
+    border-radius: 20px;
+    box-shadow: 0 7px 25px rgba(0, 0, 0, 0.08);
+    height: 275px;
+    width: 1855px;
```

We need to keep everything viewable on a single page, so we don't want to use static widths for our elements.

```suggestion
    width: 100%;
```